mocker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Github Repository Link
+GH_REPO=github.com/bakurits/mattermost-plugin-anonymous
+
+# Array of (destination,interface) tuples
+# If new element needs to be added, please delimit destination and interface by comma like other elements
+array=(
+    anonymous,Anonymous
```

Comment:
I think extra space between will make that more readable

Reply:
Space is invalid solution as it parses using space. Any delimiter will do but not space,enter or tab. We can choose delimiter which won't occur in file names or interfaces

Reply:
Then a colon would be better

Reply:
done